tests/agent/test_rum.py

```diff
@@ -0,0 +1,13 @@
+from tests.endpoint import Endpoint
+from tests import utils
+import requests
+
+
+def test_rum(rum):
+    elasticsearch = rum.apm_server.elasticsearch
+    elasticsearch.clean()
+    endpoint = Endpoint(rum.url, "run_integration_test")
```

Comment:
would it be possible to customize the response of that endpoint?  Perhaps with a query parameter?  We could use `check_agent_transaction` instead of duplicating it if this returned the text `run_integration_test` or something else static.

Reply:
I'm not sure I understand! Currently the endpoint returns the request that was made to the apm-server (including the body) and the response that was received by the agent.

We can examine this response however since we already test agent <-> apm-server communication in the agent, I thought we shouldn't duplicate it here.

Reply:
Sorry, that is indeed confusing.  This test duplicates`check_agent_transaction`, minus the `check_request_response` bit: https://github.com/elastic/apm-integration-testing/blob/46b8e044c95a638c34bc8a774e4b810b153ebe18/tests/utils.py#L23.  If the endpoint returned a predictable message, we could avoid that.

Reply:
Ah, good point! I'll change the endpoint to accept a query parameter!